applications/StructuralMechanicsApplication/custom_elements/truss_element_3D2N.cpp

```diff
@@ -442,6 +443,41 @@ void TrussElement3D2N::CalculateOnIntegrationPoints(
     if (rOutput.size() != integration_points.size()) {
         rOutput.resize(integration_points.size());
     }
+
+    if (rVariable == PRESTRESS_VECTOR){
```

Comment:
can you please make a separate PR to the StructuralMechanicsApp and add a test?

Reply:
@KlausBSautter Did you add this to be able to have a pre-stress on the membrane for the water slide case in 3D?

Reply:
I eliminated it definitively, because I talked with Klaus and this was no longer required, nor used in the PFEM coupling.